examples/threathunter/threat_intelligence/get_feed_ids.py

```diff
@@ -0,0 +1,17 @@
+from cbapi.psc.threathunter import CbThreatHunterAPI
+from cbapi.psc.threathunter.models import Feed
+import logging
+
+log = logging.getLogger(__name__)
+
+def get_feed_ids():
+    cb = CbThreatHunterAPI()
+    url = "/threathunter/feedmgr/v2/orgs/{}/feeds".format(cb.credentials.org_key)
+    feeds = cb.get_object(url)
```

Comment:
Change this get call instead to a select `cb.select(Feed)`. If you need to inject org_key, auth_tokens, and urls then use the CbThreatHunterAPI parameters to pass those in there

Reply:
Updated to cb.select()